### DISCUSSION TITLE
Laptop: ThinkPad E16 Gen 2 Intel

### DISCUSSION THREAD

**Discussion by Secret-chest** [linuxmint/discussions#536] [Hardware reviews]
### Releases

Linux Mint 22 beta

### Compatibility

Works perfectly

### Description of the hardware

Everything works. Configuration:

* Intel 155H
* Intel Wi-Fi 6E card
* 57Wh battery
* 16G RAM
* 512G SSD
* 2560×1600 display
* Backlit keyboard

However, Lm 21 doesn't support the Wi-Fi, touchpad and sound, although 22 does.

### Optional - Notes

Install `intel-media-va-driver-non-free` for better performance and compatibility.

**Comment by AtomicRobotMan0101** [comment]
simply update the Kernel. 

LM21 comes with 5.15 LTS and 22 comes with 6.8LTS

This is the problem - an easy fix.

**Reply by Secret-chest** [reply]
22 is released now though.
